web/src/buddy_list.ts

```diff
@@ -167,7 +167,7 @@ export class BuddyList extends BuddyListConf {
     $participants_list = $(this.participants_list_selector);
     $users_matching_view_list = $(this.matching_view_list_selector);
     $other_users_list = $(this.other_user_list_selector);
-    current_filter: Filter | undefined;
+    current_filter: Filter | undefined | "unset" = "unset";
```

Comment:
Is this a reasonable solution, or is there something else that's more best practices?

Reply:
I considered that, but I was worried that would be confusing to have both `undefined` and `null`, and `"unset"` is a lot more clear what it's doing.